Runtime/Data/Attribute/RestrictedAttribute.cs

```diff
@@ -14,18 +14,26 @@ public class RestrictedAttribute : PropertyAttribute
         [Flags]
         public enum Restrictions
         {
+            /// <summary>
+            /// The property is de-emphasized.
+            /// </summary>
+            Muted = 1 << 0,
             /// <summary>
             /// The property is always read-only.
             /// </summary>
-            ReadOnlyAlways = 1 << 0,
+            ReadOnlyAlways = 1 << 1,
+            /// <summary>
+            /// The property is read-only when the application is playing.
+            /// </summary>
+            ReadOnlyAtRunTime = 1 << 2,
```

Comment:
```suggestion
            ReadOnlyAtRuntime = 1 << 2,
```
https://en.wikipedia.org/wiki/Run_time

Not sure honestly, I just always read `runtime` and `design time`... (Also change the others if we decide to change this.)

Reply:
Run Time is short for Running Time which is two separate words. Runtime to me is more of a noun than a verb (state)

Run time is the state in when the program is running

Runtime is the thing that is doing the running (e.g. Java Runtime)

if that makes sense?

Reply:
It does. 👍